target/scripts/check-for-changes.sh

```diff
@@ -6,37 +6,34 @@
 # shellcheck source=./helpers/index.sh
 source /usr/local/bin/helpers/index.sh
 
-function _log_date
-{
-  date +"%Y-%m-%d %H:%M:%S"
-}
-
 function _changedetector_notify
 {
-  local LOG_LEVEL_ARGUMENT="${1}"
+  local LEVEL="${1}"
   shift 1
 
-  _notify "${LOG_LEVEL_ARGUMENT}" '[ CHANGEDETECTOR ]' " $(_log_date) " "${*}"
+  _notify "${LEVEL}" '[ CHANGEDETECTOR ]' " $(date +"%Y-%m-%d %H:%M:%S") " "${*}"
 }
```

Comment:
Is there a reason to use `shift 1` and `${*}` here instead of just `${2}`? It's expected to always be two args right? (log level and message)

It seems that you are passing `[ CHANGEDETECTOR ]` and the log date as separate values, instead of wrapping each as a single message string arg?

Reply:
`_notify' itself is written in the same way: the first argument is the log level, all the others are messages. It is therefore not actually expected to be exactly two, but at least two. This is done in order to enable one to split longer lines into multiple arguments.

However, in this case, I could indeed write these arguments as one :D

Reply:
Done in cce33f80d545c23eaddb67a4afb2a2bd9596e0d9